### PR TITLE
Handle US Locale number format when parsing xunit result file

### PR DESCRIPTION
**Pull request by vincentdo**

### CODE REVIEW

**Comment by john-m-liu** [comment]
Could you link the corresponding cucumber change that prompted this?

**Comment by vincentdo** [comment]
I'm assuming you mean on the cucumber side and not on the cloud side:
https://github.com/cucumber/cucumber-jvm/blob/v5.4.1/core/src/main/java/io/cucumber/core/plugin/JUnitFormatter.java#L83

File: command/results_xunit_parser.go [hunk 1]
```diff
@@ -15,13 +16,25 @@ import (
 	"github.com/pkg/errors"
 )
 
+type customTime float64
```
Comment (john-m-liu) [inline command/results_xunit_parser.go]: Could you rename this something like "customFloat?" It technically isn't a time

**Comment by vincentdo** [comment]
@john-m-liu addressed your comments

**Pull request merged**